### PR TITLE
Fix part of #2003: Replacing elevation_0dp, elevation_4dp and space_224dp

### PR DESCRIPTION
**Pull request by Arjupta**
## Explanation

Fixes part of #2003 This PR replaces the use of elevation_0dp , elevation_4dp and
space_224dp with the dimensions of contextful names.

## Checklist

- [x] The PR title starts with "Fix #bugnum: ", followed by a short, clear summary of the changes. (If this PR fixes part of an issue, prefix the title with "Fix part of #bugnum: ...".)
- [x] The PR explanation includes the words "Fixes #bugnum: ..." (or "Fixes part of #bugnum" if the PR only partially fixes an issue).
- [x] The PR follows the [style guide](https://github.com/oppia/oppia-android/wiki/Coding-style-guide).
- [x] The PR does not contain any unnecessary auto-generated code from Android Studio.
- [x] The PR is made from a branch that's **not** called "develop".
- [x] The PR is made from a branch that is up-to-date with "develop".
- [x] The PR's branch is based on "develop" and not on any other branch.
- [x] The PR is **assigned** to an appropriate reviewer in both the **Assignees** and the **Reviewers** sections.

### CODE REVIEW

File: app/src/main/res/layout-sw600dp-land/topic_info_fragment.xml [hunk 1]
```diff
@@ -15,7 +15,7 @@
     android:orientation="horizontal">
 
     <ScrollView
-      android:layout_width="@dimen/elevation_0dp"
+      android:layout_width="@dimen/topic_info_fragment_scroll_view_width"
```
Comment (rt4914) [inline app/src/main/res/layout-sw600dp-land/topic_info_fragment.xml]: This should actually be hardcoded `0dp` directly here. It should not be defined in `strings.xml` file.

File: app/src/main/res/layout-sw600dp-land/topic_info_fragment.xml [hunk 2]
```diff
@@ -26,7 +26,7 @@
 
         <TextView
           android:id="@+id/topic_name_text_view"
-          android:layout_width="@dimen/elevation_0dp"
+          android:layout_width="@dimen/topic_info_fragment_topic_name_text_view_width"
```
Comment (rt4914) [inline app/src/main/res/layout-sw600dp-land/topic_info_fragment.xml]: This should actually be hardcoded `0dp` directly here. It should not be defined in `strings.xml` file.

**Review by rt4914 — CHANGES_REQUESTED** [review]
Minor change suggested. Thanks.

**Comment by rt4914** [comment]
@ARJUPTA You have missed the changes that I had suggested earlier.

File: app/src/main/res/layout-sw600dp-land/topic_info_fragment.xml (hunk 1 above)
Comment (Arjupta) [reply app/src/main/res/layout-sw600dp-land/topic_info_fragment.xml]: Okay will do the same for space_0dp

File: app/src/main/res/layout-sw600dp-land/topic_info_fragment.xml (hunk 2 above)
Comment (Arjupta) [reply app/src/main/res/layout-sw600dp-land/topic_info_fragment.xml]: done and removed that dimension from dimens.xml

**Review by rt4914 — APPROVED** [review]
LGTM

**Pull request merged**